### ISSUE TITLE
Error printing zero points

### ISSUE THREAD

**Issue by christianpaquin** [paulmillr/noble-bls12-381#52]
This code fails:
```
console.log(PointG1.ZERO);
console.log(PointG2.ZERO);
```
with the following stack trace:
```
Error: invert: expected positive integers, got n=0 mod=4002409555221667393417789825735904156556882819939007885332058136124031650490837864442687629129015664037894272559787
    at invert (/node_modules/@noble/bls12-381/lib/math.js:75:15)
    at Fp.invert (/node_modules/@noble/bls12-381/lib/math.js:108:23)
    at PointG1.toAffine (/node_modules/@noble/bls12-381/lib/math.js:625:28)
    at PointG1.toString (/node_modules/@noble/bls12-381/lib/math.js:619:29)
```

Should the `ProjectivePoint.toString` function check for the ZERO point before trying to invert?

**Comment by paulmillr** [comment]
Yes

**Comment by paulmillr** [comment]
yes please

**Comment by paulmillr** [comment]
published in 1.3.0
